crates/burn-vision/src/image/agumentation.rs

```diff
@@ -0,0 +1,404 @@
+use burn_tensor::{ElementConversion, Tensor, TensorData, backend::Backend};
+use image::{ColorType, DynamicImage, RgbImage};
+use rand::Rng;
+
+/// Provides injectable traits for testing
+trait InjectableTraits {
+    fn get_random<T>(&mut self) -> T
+    where
+        rand::distr::StandardUniform: rand::distr::Distribution<T>;
+
+    fn get_random_with_range<R, T>(&mut self, range: R) -> T
+    where
+        R: rand::distr::uniform::SampleRange<T>,
+        T: rand::distr::uniform::SampleUniform;
+}
+
+struct RngRandom;
+
+impl InjectableTraits for RngRandom {
+    fn get_random<T>(&mut self) -> T
+    where
+        rand::distr::StandardUniform: rand::distr::Distribution<T>,
+    {
+        rand::rng().random::<T>()
+    }
+
+    fn get_random_with_range<R, T>(&mut self, range: R) -> T
+    where
+        R: rand::distr::uniform::SampleRange<T>,
+        T: rand::distr::uniform::SampleUniform,
+    {
+        rand::random_range(range)
+    }
+}
+
+#[derive(Clone, Debug)]
+struct Agumentations<R: InjectableTraits = RngRandom> {
+    rng: R,
+}
+
+impl Default for Agumentations {
+    fn default() -> Self {
+        Self { rng: RngRandom }
+    }
+}
+
+impl<R: InjectableTraits> Agumentations<R> {
+    #[allow(dead_code)]
```

Comment:
Why do we have `#[allow(dead_code)]` on each method? I think if you make `Agumentations` pub and export, you will not face this issue.

Reply:
Thanks for the tip! Fixed